docs/transmission.md

```diff
@@ -0,0 +1,89 @@
+0 for some time (mimicking an exposed compartment). Once infectious, agents have a pre-assigned
+number of infection attempts. Infection attempt times are drawn from the generation interval.
+Once there are no more infection attempts remaining for the agent, we argue that they do not
+transition to a recovered compartment but rather back to a susceptible compartment but
+with high immunity.
+
+## Workflow
+
+The transmission model is centered around a key workflow:
+1. Watch for an agent becoming infectious (S --> I transition).
+2. For the infectious agent, draw a number of infection attempts. This can be from the
+$R_i$ distribution or the contact distribution.
+3. Draw an infection time for the agent. This consists of taking an ordered draw from the generation
+interval. The math for this is abstracted away under a generic `get_next_infection_time` method.
+4. Draw a susceptible contact from the agent's contact group. Contacts could be drawn uniformly
+from the contact group or with weight based on person characteristics. The actual
+drawing of a contact can be abstracted into a `get_contact` method. Evaluate whether
+the infection attempt is successful. Success is based on:
+    - a potential probability of transmission in the case where we are drawing the
+    number of contacts from a contact distribution,
+    - whether any interventions are present that may reduce transmission,
+    - the contact's immunity level which is updated when the contact is drawn.
+
+    If the infection attempt is successful, the infectious contact is labeled as infectious.
+5. Repeat steps #3 and #4, decreasing the number of infection attempts left by one each time
+regardless of whether the attempt is successful and ending when the agent has no zero
+infection attempts remaining.
+6. Re-label the agent as susceptible. The immunity manager will set their latent immunity level
+to the value of immunity based on natural infection to mimic a recovered compartment.
+
+## Relationship to other managers/out of scope for the transmission manager
+
+- The transmission manager does not implement interventions. It does not keep track
+of interventions present or how they impact transmission. Instead, those quantities are
+stored in an intervention plugin. The transmission manager can query an individual's transmissibility
+at a given time based on their person characteristics to get their realized transmissibility
+for use in the workflow described above, but it does not explicitly track which interventions
+are present at what time or how they impact transmissibility.
+
+- The transmission manager also does not track an individual's contacts. Instead, the transmission
+manager calls a generic method called `get_contact` which is in the `mod contact_manager` to obtain
+the individual's contacts.
+
+- The transmission manager does not track an individual's health status. In fact, it is
+completely independent from health status. Instead, health status is tracked in
```

Comment:
We can remove this as well. I tihnk it's enough for now to say what transmission does

Reply:
I would argue that these components are important to include so that we don't build other elements of transmission potential (contact manager) or disease progression into the manager. Ok to decrease detail but I think part of "what the transmission manager does" is specifying what it doesn't do